pkg/machinepool/machinepool.go

```diff
@@ -948,6 +952,43 @@ func (m *machinePool) CreateNodePools(r *rosa.Runtime, cmd *cobra.Command, clust
 		}
 	}
 
+	sumOfReplicas := replicas
```

Comment:
all these changes should only work for HCP, but not for classic.

Reply:
They are placed in the `CreateNodePools` and `EditNodePool` functions, which are ran only for HCP